src/applications/disability-benefits/all-claims/pages/alternateNames.js

```diff
@@ -0,0 +1,66 @@
+import FullNameField from 'us-forms-system/lib/js/fields/FullNameField';
+
+export const uiSchema = {
+  'view:hasAlternateName': {
+    'ui:title': 'Did you serve under another name?',
+    'ui:widget': 'yesNo'
+  },
+  alternateNames: {
+    'ui:description': 'What names did you serve under?',
+    'ui:options': {
+      viewField: FullNameField,
+      reviewTitle: 'Other names',
+      expandUnder: 'view:hasAlternateName'
```

Comment:
You could also include `addAnotherLabel` and set it to `Add Another Name` to change the add another button label. That's usually something the content folks ask for in their review on other forms I've worked on.

Reply:
Ah! Thanks, didn't know that was a thing.